transport-native-epoll/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -428,4 +434,5 @@ private static native DatagramSocketAddress recvFromAddress(
     private static native void setSoLinger(int fd, int soLinger) throws IOException;
     private static native void setTcpDeferAccept(int fd, int deferAccept) throws IOException;
     private static native void setTcpQuickAck(int fd, int quickAck) throws IOException;
+    private static native int peerCredentials(int fd, PeerCredentials pc);
```

Comment:
once you use `netty_unix_socket_getOption` this method should have `throws IOException;` like other native socket option accessor methods